### PR TITLE
feat: SOF-1090 Add Graphics Setups to Offtube showstyle

### PR DESCRIPTION
**Pull request by ianshade**
Adds the table, changes properties names, migrates

### CODE REVIEW

File: src/tv2_afvd_showstyle/migrations/index.ts [hunk 1]
```diff
@@ -233,6 +234,13 @@ export const showStyleMigrations: MigrationStepShowStyle[] = [
 	 */
 	removeSourceLayer('1.7.5', 'AFVD', 'studio0_graphicsIdent_persistent'),
 
+	/**
+	 * 1.7.6
+	 * - Rename columns in Graphics Setups table
+	 */
+	renameTableColumn('1.7.6', 'GraphicsSetups', 'OvlShowName', 'OvlShowName'),
```
Comment (RasmusAlbrektsen) [inline src/tv2_afvd_showstyle/migrations/index.ts:241]: Why do these use the same value for "old" and "new", am i missing something? :-)

Comment (ianshade) [reply src/tv2_afvd_showstyle/migrations/index.ts]: My mistake, thanks!

File: src/tv2-common/blueprintConfig.ts [hunk 2]
```diff
@@ -39,10 +39,8 @@ export interface TableConfigSchema {
 
 export interface TableConfigGraphicsSetup {
 	Name: string
-	VcpConcept: string
-	OvlShowId: string
-	FullShowId: string
-	DveLayoutFolder: string
+	OvlShowName: string
+	FullShowName?: string
```
Comment (LindvedKrvang) [inline src/tv2-common/blueprintConfig.ts:43]: What is the reason for this being optional? 
Isn't this kinda the same as the `iNewsColumnStyle` from Rasmus PR?

File: src/tv2_offtube_showstyle/helpers/config.ts [hunk 3]
```diff
@@ -36,20 +36,21 @@ export interface DVEConfigInput {
...
-	_config: TV2ShowstyleBlueprintConfigBase
-): TableConfigGraphicsSetup {
-	// just for type compatibility, not really supported in offtube
-	return {
-		Name: '',
-		VcpConcept: '',
-		OvlShowId: '',
-		FullShowId: '',
-		DveLayoutFolder: ''
+function findGraphicsSetup(context: ICommonContext, config: OfftubeShowStyleConfig): TableConfigGraphicsSetup {
```
Comment (LindvedKrvang) [inline src/tv2_offtube_showstyle/helpers/config.ts:42]: This method seems very similar to the `findGraphicsSetup` for Galleries. It seems it is the type of the parameter config that differs. Couldn't we extract the two methods so we don't have this logic twice? Or does `OfftubeShowStyleConfig` not have anything in commen with the gallery version?

File: src/tv2-common/helpers/graphics/viz/index.ts [hunk 4]
```diff
@@ -59,6 +59,26 @@ export function GetInternalGraphicContentVIZ(
 	}
 }
 
+export function findShowName(
```
Comment (LindvedKrvang) [inline src/tv2-common/helpers/graphics/viz/index.ts:62]: It appears that this method is only used in here. If that is true, then I think we shouldn't export

**Review by LindvedKrvang — APPROVED** [review]

File: src/tv2-common/blueprintConfig.ts (hunk 2 above)
Comment (ianshade) [reply src/tv2-common/blueprintConfig.ts:43]: This is due to how the types and Viz graphics are structured - they currently are in tv2-common and we still support Viz graphics in both Show Styles, but only the Gallery blueprints expose the `FullShowName`. We spoke about removing that cross-support and sticking with Viz in Gallery (AFVD) Show Style and HTML in Qbox (Offtube) Show Style, but if does not seem to be a part of this task.

**Review by KvelaGorrrrnio — APPROVED** [review]

File: src/tv2-common/helpers/graphics/viz/index.ts (hunk 4 above)
Comment (ianshade) [reply src/tv2-common/helpers/graphics/viz/index.ts]: I removed `export`

File: src/tv2_offtube_showstyle/helpers/config.ts (hunk 3 above)
Comment (ianshade) [reply src/tv2_offtube_showstyle/helpers/config.ts]: I replaced it with a generic implementation in tv2-common

**Review by LindvedKrvang — APPROVED** [review]

**Pull request merged**